examples/with-mongodb/util/mongodb.js

```diff
@@ -0,0 +1,20 @@
+import { MongoClient } from 'mongodb'
+
+console.log(process.env.MONGODB_URI)
+let uri = process.env.MONGODB_URI
+let cachedDb = null
+
```

Comment:
Add a simple validation here for the mongo url that will make it fail at build time, like this:

```jsx
if (!process.env.MONGODB_URI) {
  throw new Error('Please define the MONGODB_URI environment variable inside .env.local')
} 
```

Reply:
Sounds good